.ci/deps.sh

```diff
@@ -55,8 +55,10 @@ done
 python setup.py --help
 
 # Dart Lint commands
-wget -nc -O ~/dart-sdk.zip https://storage.googleapis.com/dart-archive/channels/stable/release/1.14.2/sdk/dartsdk-linux-x64-release.zip
-unzip -n ~/dart-sdk.zip -d ~/
+if [ ! -d "~/dart-sdk/" ] && [ ! -d "~/dart-sdk/bin" ]; then
```

Comment:
however, i think it would have a conflict with that? echo 'export PATH=$PATH:~/dart-sdk/bin' >> ~/.circlerc